CDP4WspDatabaseAuthentication/Cdp4WspDatabaseAuthenticatorConnector.cs

```diff
@@ -0,0 +1,104 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Cdp4WSPDatabaseAuthenticatorConnector.cs" company="RHEA System S.A.">
+//   Copyright (c) 2016 RHEA System S.A.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4WspDatabaseAuthentication
+{
+    using CDP4Authentication;
+    using CDP4Authentication.Contracts;
+
+    /// <summary>
+    /// A connector for WSP basic authentication against a CDP4 database.
+    /// </summary>
+    public class Cdp4WspDatabaseAuthenticatorConnector : AuthenticatorConnector<AuthenticatorProperties>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Cdp4WspDatabaseAuthenticatorConnector"/> class.
+        /// </summary>
+        /// <param name="properties">
+        /// The properties.
+        /// </param>
+        public Cdp4WspDatabaseAuthenticatorConnector(AuthenticatorProperties properties)
+            : base(properties)
+        {
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether the connector is up.
+        /// </summary>
+        public override bool IsUp
+        {
+            get
+            {
+                this.IsDown = false;
```

Comment:
this is a bit tricky i think. Here you do something with IsUp and as a side effect you cange IsDown, but you don't have similiar logic for the IsDown property? Out of curiosity, when is this down?

Reply:
you close the conversation, but no response and the code seems to be the same?

Reply:
IsDown property is always false in both connectors. For the moment it seems to not be used. Did those properties IsUp & IsDown have been used in the past ?